packages/react-core/src/components/DescriptionList/DescriptionListText.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
```

Comment:
I actually don't think we need this component.  I think it can be rendered internally within the `DecriptionListTerm` and `DescriptionListDescription`. 
@mattnolting  will Should everything rendered in the `<dt>` and the `<dd>` be wrapped in the `.pf-c-description-list__text` `<div>`?

Reply:
cc @mcoker we added the `__text` element for future requests, like truncation, wrapping etc. I'd expect it to be a `span` though. @mcoker wdyt?

Reply:
@mattnolting yep, agree

Reply:
@mattnolting @tlabaj should definitely be a span in the `<dt>` so it can be displayed inline with things around it and those thing wrap like content in form labels. The basic layout will be similar to a form.

<img width="171" alt="Screen Shot 2020-07-21 at 2 28 58 PM" src="https://user-images.githubusercontent.com/35148959/88098225-0ab22480-cb5f-11ea-98b4-cee4fca5efdd.png">


@tlabaj yep I would expect the children of the `dt` and `dd` to go in the `__text` element. We may enable some prop on them to create elements we will put adjacent to the `__text` element - for example in the `dt`, a prop that will generate a help icon with tooltip/popover text.